### DISCUSSION TITLE
Depricate request [@types/stripe-v3]

### DISCUSSION THREAD

**Discussion by kuncevic** [DefinitelyTyped/DefinitelyTyped#64046] [Issues with a @types package]
The current @types/stripe-v3 is 3.1.27 however a while ago stripe started shipping typescript types as a part of an official [package]( https://github.com/stripe/stripe-js/tree/master/types).

Recently I've was just being confused by using the v 3.1.27 of the @types/stripe-v3 to start with. Then by fiddling around realized had to use official types from @stripe/stripe-js because @types/stripe-v3 types does not match the latest stripe API version

[a comment by an automated account is not shown]

**Comment by amritk** [comment]
Sounds good to me. It's about time!

**Comment by kaznovac** [comment]
I've tried removing these types a while ago (when stripe released the official types) the problem is that some of the (now mostly deprecated) libraries depend on these @types...

To do this transition correctly the depending libraries should be adapted to use the official types first, at the time I've was considering this there was more work required than I could lend...

**Reply by kuncevic** [reply]
May be at least deprecation message with a link to the latest in README.md would be a great way to start with

**Reply by kaznovac** [reply]
Sure, you are welcome to create a PR :)

**Comment by ksol** [comment]
I tried to take a stab at removing this package, but either I don't understand how to properly use `npm run not-needed`, either it's not well suited if the types package version does not follow the same versions number as the regular lib package 🤔

**Comment by ghost** [comment]
Created PR: #68416
